templates/gutterball.conf.erb

```diff
@@ -10,5 +10,6 @@ jpa.config.hibernate.show_sql=false
 # password comes from /etc/pki/katello/keystore_password-file
 gutterball.amqp.keystore_password=<%= keystore_password %>
 gutterball.amqp.truststore_password=<%= keystore_password %>
+gutterball.amqp.connect=amqp://guest:guest@<%= @fqdn %>/test:5671?brokerlist='tcp://<%= @fqdn %>:5671'?ssl='true'&ssl_cert_alias='gutterball'
```

Comment:
gutterball.amqp.connect=amqp://guest:guest@<%= @fqdn %>/test:5671?brokerlist='tcp://<%= @fqdn %>:5671?ssl='true'&ssl_cert_alias='gutterball''

Check ', I corrected above.

Is port 5671 always that or is it configurable somewhere? (ie. is hardcode alright?)